flask_table/table.py

```diff
@@ -69,14 +69,18 @@ def __init__(self, items, classes=None, thead_classes=None,
         self.table_id = table_id
         self.border = border
 
+    def get_border(self):
+        if self.border:
+            return ' border="1"'
+        else:
+            return ''
+
```

Comment:
I would rather you reverted this change. I've been meaning to change around the way that the table-level attributes work (see #33) and this quite separate to this issue.